dashboard/test/factories.rb

```diff
@@ -1,4 +1,8 @@
-FactoryGirl.define do
+FactoryGirl.define do  factory :survey_result do
+    user nil
+properties "MyText"
+  end
```

Comment:
Lint/BlockAlignment: `end` at 4, 2 is not aligned with `factory :survey_result do` at 1, 23 or `FactoryGirl.define do  factory :survey_result do` at 1, 0.